trello/trello_service.py

```diff
@@ -95,4 +96,34 @@ def move_card_to_new_list(card_id, to_list_name):
     response = requests.put(url, params = params)
     card = response.json()
 
-    return card
\ No newline at end of file
+    return card
+
+def create_trello_board():
+    url = "https://api.trello.com/1/boards/"
+
+    extra_params = { 'name': 'To-Do App' }
+    params = get_auth_params()
+    params.update(extra_params)
+    print(params)
+    response = requests.request(
+        "POST",
+        url,
+        params = params
+    )
+
+    print(response.text)
```

Comment:
Similar to the commented out code, we should delete the print statements before we merge this into master

Reply:
Will commit in new PR